starter-code/public/scripts/controllers/articleController.js

```diff
@@ -3,10 +3,13 @@ var app = app || {};
 
 (function(module) {
   const articleController = {};
-  // TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
+  // DONE: Setup a function that kicks off the fetching and rendering of articles, using the same
   // code that used to be in index.html.
   // Also be sure to hide all the main section elements, and reveal the #articles section:
-
+  articleController.initArticleSection = function() {
```

Comment:
You are missing one part of the above TODO: 
Setup a function that kicks off the fetching and rendering of articles, using the same code that used to be in index.html.
Here we need to call the function that fetches(HINT HINT :) with the callback we had in the index page previous.